rebasebot/bot.py

```diff
@@ -97,7 +98,7 @@ def _needs_rebase(gitwd, source: GitHubBranch, dest: GitHubBranch):
         for branch in branches_with_commit.splitlines():
             # Must strip the branch name as git branch adds an indent
             if branch.lstrip() == dest_branch:
-                logging.info("Dest branch already contains all latest changes.")
+                logging.info("Dest branch already contains all the latest changes.")
```

Comment:
I would actually drop `all` from this, it flows better as `Destination branch already contains the latest changes`

Reply:
👍